README.md

```diff
@@ -1,62 +1,47 @@
-[![GitPOAP Badge](https://public-api.gitpoap.io/v1/repo/quais-io/quais.js/badge)](https://www.gitpoap.io/gh/quais-io/quais.js)
-[![Twitter Follow](https://img.shields.io/twitter/follow/ricmoo?style=social)](https://twitter.com/ricmoo)
 
 ---
 
-A complete, compact and simple library for Ethereum and ilk, written
+A complete, compact and simple library for Quai and Qi, written
 in [TypeScript](https://www.typescriptlang.org).
 
 **Features**
 
-- Keep your private keys in your client, **safe** and sound
-- Import and export **JSON wallets** (Geth, Parity and crowdsale)
-- Import and export BIP 39 **mnemonic phrases** (12 word backup phrases) and **HD Wallets** (English as well as Czech, French, Italian, Japanese, Korean, Simplified Chinese, Spanish, Traditional Chinese)
-- Meta-classes create JavaScript objects from any contract ABI, including **ABIv2** and **Human-Readable ABI**
-- Connect to Ethereum nodes over [JSON-RPC](https://github.com/ethereum/wiki/wiki/JSON-RPC), [INFURA](https://infura.io), [quaiscan](https://quaiscan.io), [Alchemy](https://alchemyapi.io), [Ankr](https://ankr.com) or [MetaMask](https://metamask.io)
-- **Small** (~144kb compressed; 460kb uncompressed)
-- **Tree-shaking** focused; include only what you need during bundling
-- **Complete** functionality for all your Ethereum desires
-- Extensive [documentation](https://docs.quais.org/v6/)
-- Large collection of **test cases** which are maintained and added to
-- Fully written in **TypeScript**, with strict types for security and safety
-- **MIT License** (including ALL dependencies); completely open source to do with as you please
+-   Keep your private keys in your client, **safe** and sound
+-   Import and export BIP 39 **mnemonic phrases** (12 word backup phrases) and **HD Wallets** (English as well as Czech, French, Italian, Japanese, Korean, Simplified Chinese, Spanish, Traditional Chinese)
+-   Meta-classes create JavaScript objects from any contract ABI, including **ABIv2** and **Human-Readable ABI**
+-   Connect to Quai nodes over [JSON-RPC](https://qu.ai/docs/develop/apis/json-rpc-api/), [quaiscan](https://quaiscan.io), or [Pelagus](https://pelaguswallet.io)
+-   **Small** (~136kb compressed; 460kb uncompressed)
+-   **Tree-shaking** focused; include only what you need during bundling
+-   **Complete** functionality for all your Ethereum desires
+-   Extensive documentation **coming soon**
+-   Large collection of **test cases** which are maintained and added to
+-   Fully written in **TypeScript**, with strict types for security and safety
+-   **MIT License** (including ALL dependencies); completely open source to do with as you please
 
 ## Keep Updated
 
-For advisories and important notices, follow [@quaisproject](https://twitter.com/quaisproject)
-on Twitter (low-traffic, non-marketing, important information only) as well as watch this GitHub project.
+For advisories and important notices, follow [@quainetwork](https://twitter.com/quaisnetwork)
```

Comment:
`https://twitter.com/quaisnetwork` should be `https://twitter.com/quainetwork`

Reply:
Ah yes thx, fixed here: [886e374](https://github.com/dominant-strategies/quais-6.js/pull/167/commits/886e374e04bb82efd2b66cbd33f07bc1a03c3edb)